src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,10 @@
+import { UserInfo } from '../UserInfo';
+
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo todo={todo} />
```

Comment:
You are passing the entire 'todo' object as a prop to the 'UserInfo' component. It would be better to pass only the user object if the 'todo' object has a user attached to it to make the props more specific and self-descriptive.

Reply:
i will fix it, thanks
it happend because i forgot run test before deploy